articles/how-to-make-an-image-carousel-for-your-website/index.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: engineering-education
+status: publish
+published: true
```

Comment:
This paragraph is normally done by the section team just to avoid confusion. You can do away with it for now. It will be added once your article is ready to be publised.